src/api/tap.rs

```diff
@@ -168,7 +169,14 @@ impl TapController {
     pub fn new(watch_rx: WatchRx, tap_tx: TapSender, patterns: TapPatterns) -> Self {
         let (_shutdown, shutdown_rx) = oneshot::channel();
 
-        tokio::spawn(tap_handler(patterns, tap_tx, watch_rx, shutdown_rx));
+        tokio::spawn(
+            tap_handler(patterns, tap_tx, watch_rx, shutdown_rx).instrument(error_span!(
+                "tap_handler",
+                component_kind = "sink",
+                component_id = "_tap", // It isn't clear what the component_id should be here other than "_tap"
+                component_type = "tap",
+            )),
+        );
```

Comment:
Do we actually care about instrumenting the tap handler, given that it's always set to drop the newest event under backpressure scenarios? Basically, if we never expect it to be a contributor towards _upstream_ backpressure, why bother instrumenting it?

(This also avoids multiple taps getting merged together with the same component ID.)

Reply:
I have recurring nightmares about `tap` causing backpressure, so I like the idea of still measuring just in case.